<ui version="4.0" >
 <class>ParametersPage</class>
 <widget class="QWidget" name="ParametersPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameters</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Fractal Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="editType" >
       <property name="text" >
        <string/>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonType" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelJulia" >
     <property name="text" >
      <string>Julia Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelJuliaX" >
       <property name="text" >
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DoubleEdit" name="editJuliaX" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelJuliaY" >
       <property name="text" >
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DoubleEdit" name="editJuliaY" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelPosition" >
     <property name="text" >
      <string>View Position</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelPositionX" >
       <property name="text" >
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DoubleEdit" name="editPositionX" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPositionY" >
       <property name="text" >
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DoubleEdit" name="editPositionY" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelZoom" >
     <property name="text" >
      <string>Zoom Factor</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinZoom" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="prefix" >
        <string>10^</string>
       </property>
       <property name="minimum" >
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>15.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelAngle" >
     <property name="text" >
      <string>Rotation Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinAngle" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix" >
        <string> deg</string>
       </property>
       <property name="decimals" >
        <number>1</number>
       </property>
       <property name="minimum" >
        <double>0.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDefault" >
     <property name="text" >
      <string>Default Position</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore" >
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelBookmarks" >
     <property name="text" >
      <string>Bookmarks</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>84</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLoad" >
       <property name="text" >
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave" >
       <property name="text" >
        <string>Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>doubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editJuliaX</tabstop>
  <tabstop>editJuliaY</tabstop>
  <tabstop>editPositionX</tabstop>
  <tabstop>editPositionY</tabstop>
  <tabstop>spinZoom</tabstop>
  <tabstop>spinAngle</tabstop>
  <tabstop>buttonRestore</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
